public/css/customFonts.css

```diff
@@ -0,0 +1,31 @@
+@font-face {
+    font-family: 'thumbsOk';
+    src:    url('thumbsOk.eot?yd4z7j');
+    src:    url('thumbsOk.eot?yd4z7j#iefix') format('embedded-opentype'),
+        url('thumbsOk.ttf?yd4z7j') format('truetype'),
+        url('thumbsOk.woff?yd4z7j') format('woff'),
+        url('thumbsOk.svg?yd4z7j#thumbsOk') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+i {
+    /* use !important to prevent issues with browser extensions that change fonts */
+    
+    speak: none;
+    font-style: normal;
+    font-weight: normal;
+    font-variant: normal;
+    text-transform: none;
+    line-height: 1;
+
+    /* Better Font Rendering =========== */
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-thumbs-ok:before {
+    font-family: 'thumbsOk' !important;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2177439157/issues/source?bid=3148782&fileBranchId=3163784#l28)